src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.cs

```diff
@@ -497,6 +498,91 @@ public static IEnumerable<object[]> GetAsync_IPBasedUri_Success_MemberData()
             Assert.True(connectionAccepted);
         }
 
+        [Theory]
+        [InlineData("321.123.456.789")]
+        [InlineData("321.123.456.789:8080")]
+        [InlineData("[::1234]")]
+        [InlineData("[::1234]:8080")]
+        public async Task ProxiedIPAddressRequest_NotDefaultPort_CorrectlyFormatted(string host)
+        {
+            string ipAddress = "http://" + host;
```

Comment:
"ipAddress" is confusing; call this "uri" or something like that.

Reply:
Will change.